### ISSUE TITLE
Display the kind of error when loading fails

### ISSUE THREAD

**Issue by BrainStone** [lucko/LuckPermsWeb#38]
The kind of error should be displayed. To allow to differentiate between rate limit errors and typos. If an error occurs the JSON document contains an error message. If no such message could be found it should say "Generic Error".

**Comment by Turbotailz** [comment]
I agree the error handling needs improving. I am not sure on all the possible errors, though.
